dist/index.js

```diff
@@ -27,15 +27,13 @@ function PrismPlugin() {
     var optsParam = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};
 
     var opts = new _options2.default(optsParam);
-    var document = null;
```

Comment:
@ivanaveliskova Do you remember what the document + the check below removal is for? Just want to make sure I understand that change

Reply:
good question. I think that change came from when I was copy/pasting from gitbook's slate prism repo to whatever the last version they had available: https://github.com/GitbookIO/slate-prism. Since originally in guru-ui we were using the npm version, the first commit here is where this change happens which is basically getting it equivalent to the version that was being used in guru-ui

Reply:
I think the essential commit here is the second one where we update to 0.39. Since the first commit just makes this equivalent to what was being used in guru-ui already where those changes were already being used by guru-ui.

Reply:
Awesome, thank you!